### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by cortner** [cortner/QMMM2.jl#1]
- [x] Debug `d2Es` training - or explain why it doesn't converge
- [X] Regularisation : CO
- [x] Accuracy test on training set : CO
- [x] Convergence Tests : CO
- [ ] think about test sets : CO & HC
- [ ] Lattice relaxation: HC
   - comparison of gap and locality for relaxed / unrelaxed
   - create training set for relaxed
- [ ] force-based training data: HC (no rush)
- [ ] virial-based training data: CO & HC (no rush)
   - first need TB virials
- [ ] implement training of d3Es
- [ ] Implement Energy-mixing + ghost-force test
- [ ] Implement a point defect test problem
